DOC-2446: Add TINY-10374 release note entry (#3327)
tinymce/tinymce-docs: modules/ROOT/pages/7.2-release-notes.adoc

[[improvements]]
== Improvements

{productname} {release-version} also includes the following improvements:

=== Added `referrerpolicy` as a valid attribute for an iframe element.
// #TINY-10374

> Previously in {productname}, the `referrerpolicy` attribute was not a valid attribute for an iframe element. In {productname} {release-version}, the `referrerpolicy` attribute has been added to the schema as a valid attribute for an iframe element. This allows users to add an iframe with a `referrerpolicy` attribute.

Reviewer: ```suggestion
Previously in {productname}, the `referrerpolicy` attribute was not considered as valid for iframe elements. In {productname} {release-version}, this attribute has been added to the schema, allowing users to include the `referrerpolicy` attribute in iframes.
```
